### ISSUE TITLE
Error while trying to use authenticate using mock oidc server

### ISSUE THREAD

**Issue by tapajyotiD** [HarryMWinters/fastapi-oidc#19]
I am trying to add OIDC by following your example in the documentation. I am using a docker container for mock-oidc server with the following env variables: 

```
      - PORT=9090
      - CLIENT_ID=my-client
      - CLIENT_SECRET=my-secret
      - CLIENT_REDIRECT_URI=http://localhost:8000/protected
      - CLIENT_LOGOUT_REDIRECT_URI=http://localhost:8000/logout
```

in main.py I have this - 
```
OIDC_config = {
    "client_id": os.getenv("CLIENT_ID"), 
    "base_authorization_server_uri": "http://localhost:9090/.well-known/openid-configuration",
    "issuer": "http://localhost:9090/.well-known/openid-configuration",
    "signature_cache_ttl": 3600,
}

authenticate_user: Callable = get_auth(**OIDC_config)


@app.get("/protected")
def protected(id_token: IDToken = Depends(authenticate_user)):
    return {"Hello": "World", "user_email": id_token}
```
On clicking authorize, its throwing me redirect uri not matching. What could be the issue. Also, in the scope **OpenIdConnect (OAuth2, refresh_token)**, its throwing client id is invalid. I am not sure why this is happening.
![image](https://user-images.githubusercontent.com/20902191/128843266-476577d1-ab50-439b-826d-16fbe75d3b82.png)
![image](https://user-images.githubusercontent.com/20902191/128843583-a31a738b-c721-4c1e-a5d4-aeb686f53e2d.png)

Could I somehow not land in the list of api's page in the first place. Basically I would want user to first get authenticated and then the list of api's would be shown. Thanks in advance :)

**Comment by HarryMWinters** [comment]
Hey @tapajyotiD :) Thanks for posting. 

I probably won't have time to thoroughly investigate this for a couple of weeks but my recollection is that the Swagger/Open-API-3 auto-docs are broken for OIDC.  [This](https://github.com/swagger-api/swagger-ui/issues/5473) issue claims support is implemented though so maybe there's a bug in `fastapi_oidc` or maybe fastapi hasn't pulled in the latest swagger yet?

In either case, the Swagger docs and client are coming from FastAPI so we'll probably have to address it there but I'll need to do more research.

**Comment by HarryMWinters** [comment]
Closed in #24
